crates/database/src/onchain_broadcasted_orders.rs

```diff
@@ -13,7 +13,6 @@ pub enum OnchainOrderPlacementError {
     DisabledOrderClass,
     ValidToTooFarInFuture,
     InvalidOrderData,
-    InsufficientFee,
```

Comment:
Hm, wonder If I am allowed to delete the variant (for backward compatibility with older orders that might have this error)

Reply:
You are right, this would probably lead to issues if the order is read from disk

Reply:
I reverted the bare minimum of `InsufficientFee` in database so that, when we fetch old orders that have this error, nothing breaks.